src/components/SmallCard/SmallCard.js

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components/macro'
+
+export default function SmallCard({ name, images = [''] }) {
+  return (
+    <CardContainer>
+      <h2>{name}</h2>
+      {images.map(image => (
+        <img key={image} src={image} alt="" width="100" height="auto" />
+      ))}
+    </CardContainer>
+  )
+}
+
+const CardContainer = styled.section`
+  background-color: #ffffff;
```

Comment:
Small Tipp:
`#fff` is equal to `#fffff`
You could also consider using a `:root`selector globally, to manage your colors :)